lib/api_browser/app/js/factories/template_for.js

```diff
@@ -80,6 +80,20 @@ app.provider('templateFor', function() {
     }
   });
 
+  this.register(function homePageResolver($family, $requestedTemplate) {
+    'ngInject';
+    if ($requestedTemplate === 'home') {
+      switch ($family) {
```

Comment:
yes, changed the file naming to follow the family. the if/case statement can probably be carefaully simplified to be based on family.